homeassistant/components/transmission/__init__.py

```diff
@@ -102,9 +107,17 @@
     )
 )
 
+CONFIG_SCHEMA = cv.removed(DOMAIN, raise_if_present=False)
```

Comment:
This is no longer needed, we automatically raise a repair issue nowadays.

```suggestion
```

Reply:
I had to add this because I am using `async_setup`. If I remove it I am getting errors.

Reply:
I think you can use this instead
`CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)`